shallow_backup/git_wrapper.py

```diff
@@ -92,6 +93,13 @@ def move_git_repo(source_path, new_path):
 	"""
 	Moves git folder and .gitignore to the new backup directory.
 	"""
+	new_git_dir = os.path.join(new_path, '.git')
+	new_git_ignore = os.path.join(new_path, '.gitignore')
+	if os.path.exists(new_git_dir) or os.path.exists(new_git_ignore):
```

Comment:
Of these two options, I think the first one would be preferred, although maybe it might make sense to add a bit of additional detail into the error message, like:

> Evidence of a git repo has been detected

Then either one of these (or both ?):
> * A .git directory was found inside: {path}

or
> * A .gitignore file was found inside: {path}

Reply:
Yeah, that makes sense. With the extra detail in the error message, we solve the issue with the first option. Let's go with that.

`Evidence of a git repo has been detected.`

Followed by a more descriptive error message, like you suggested.

All relevant errors should be printed:

1. `A .git repo was found at: {path}`
2. `A .gitignore file was found at: {path}` 

And then print an "Exiting to prevent accidental deletion of user data." message.